providers/curation/github.js

```diff
@@ -271,80 +300,38 @@ ${this.formatDefinitions(patch.patches)}`
-   * @returns {[URL]} - Array of URLs describing the available curations
+   * @returns {[EntityCoordinates]} - Array of coordinates describing the available curations
    */
   async list(coordinates) {
-    await this.ensureCurations()
-    const root = `${this.tempLocation.name}/${this.options.repo}/${this._getSearchRoot(coordinates)}`
-    if (!fs.existsSync(root)) return []
-    return new Promise((resolve, reject) => {
-      const result = []
-      readdirp({ root, fileFilter: '*.yaml' })
-        .on('data', entry => result.push(...this.handleEntry(entry)))
-        .on('end', () => resolve(result))
-        .on('error', reject)
-    })
-  }
-
-  handleEntry(entry) {
-    const curation = yaml.safeLoad(fs.readFileSync(entry.fullPath.replace(/\\/g, '/')))
-    const { coordinates: c, revisions } = curation
-    const root = `${c.type}/${c.provider}/${c.namespace || '-'}/${c.name}/`
-    return Object.getOwnPropertyNames(revisions).map(version => root + version)
-  }
-
-  async ensureCurations() {
-    if (this.curationUpdateTime && Date.now() - this.curationUpdateTime < this.options.curationFreshness) return
-    const { owner, repo } = this.options
-    const url = `https://github.com/${owner}/${repo}.git`
-    this.tempLocation = this.tempLocation || tmp.dirSync(this.tmpOptions)
-    // if the location does not exist (perhaps it got deleted?), create it.
-    return new Promise((resolve, reject) => {
-      if (!fs.existsSync(this.tempLocation.name)) {
-        this.tempLocation = tmp.dirSync(this.tmpOptions)
-        // if it's still not there bail. Perhaps permissions problem
-        if (!fs.existsSync(this.tempLocation.name)) reject(new Error('Curation cache location could not be created'))
-      }
-      const command = this.curationUpdateTime
-        ? `cd ${this.tempLocation.name}/${repo} && git pull`
-        : `cd ${this.tempLocation.name} && git clone ${url}`
-      this.curationUpdateTime = Date.now()
-      exec(command, (error, stdout) => {
-        if (error) {
-          this.curationUpdateTime = null
-          return reject(error)
-        }
-        resolve(stdout)
-      })
-    })
+    return this.store.list(coordinates)
   }
 
   getCurationUrl(number) {
     return `https://github.com/${this.options.owner}/${this.options.repo}/pull/${number}`
   }
 
-  async getPrFiles(number) {
+  // get the list of files changed in the given PR.
+  async _getPrFiles(number) {
     const { owner, repo } = this.options
-    const github = Github.getClient(this.options)
     try {
-      const response = await github.pullRequests.getFiles({ owner, repo, number })
+      const response = await this.github.pullRequests.getFiles({ owner, repo, number })
       return response.data
     } catch (error) {
-      // @todo add logger
-      throw error
+      if (error.code === 404) throw new Error(`Could not find pr#${number} in ${owner}/${repo}`)
+      throw new Error(`Error calling GitHub to get pr#${number}. Code ${error.code}`)
```

Comment:
might want to add the owner/repo in this message too

```suggestion
      throw new Error(`Error calling GitHub to get pr#${number} in ${owner}/${repo}. Code ${error.code}`)
```

Reply:
there is only one repo